### PR TITLE
Adds the ability to fetch map data.

### PR DESCRIPTION
**Pull request by coltin**
Currently hardcoded maps.

### CODE REVIEW

File: src/main/java/com/ace/alfox/api/MapApi.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class MapApi {
+
+  @Autowired private Database playerDatabase;
+
+  @GetMapping("/api/getMap")
```
Comment (xori) [inline src/main/java/com/ace/alfox/api/MapApi.java]:
```suggestion
  @GetMapping("/api/map")
```
nit: probably don't need to repeat the verb used for fetching.

**Review by xori — APPROVED** [review]
Straightforward. Easy to recommend as we now have a structure to move forward with.

File: src/main/java/com/ace/alfox/api/MapApi.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    } else {
+      return ResponseEntity.ok(
+          "{\"zoneId\":2,\"zoneName\":\"Calcite Septum\",\"width\":4,\"height\":4,\"tiles\":[{\"tileId\":0,\"x\":0,\"y\":0,\"type\":\"plains\",\"actions\":[]},{\"tileId\":1,\"x\":1,\"y\":0,\"type\":\"plains\",\"actions\":[]},{\"tileId\":2,\"x\":2,\"y\":0,\"type\":\"plains\",\"actions\":[]},{\"tileId\":3,\"x\":1,\"y\":1,\"type\":\"plains\",\"actions\":[]},{\"tileId\":4,\"x\":2,\"y\":1,\"type\":\"water\",\"actions\":[]},{\"tileId\":5,\"x\":3,\"y\":1,\"type\":\"water\",\"actions\":[]},{\"tileId\":6,\"x\":2,\"y\":2,\"type\":\"forest\",\"actions\":[]},{\"tileId\":7,\"x\":0,\"y\":3,\"type\":\"water\",\"actions\":[]},{\"tileId\":8,\"x\":1,\"y\":3,\"type\":\"water\",\"actions\":[]},{\"tileId\":9,\"x\":2,\"y\":3,\"type\":\"forest\",\"actions\":[]},{\"tileId\":10,\"x\":3,\"y\":3,\"type\":\"forest\",\"actions\":[]}]}");
+    }
+  }
+
+  @GetMapping("/api/player")
+  public ResponseEntity setZoneId(HttpServletRequest request, @RequestParam Integer zoneId) {
```
Comment (xori) [inline src/main/java/com/ace/alfox/api/MapApi.java:40]: This function's on my hit list. 🎯🔫

File: src/main/java/com/ace/alfox/api/MapApi.java (hunk 1 above)
Comment (coltin) [reply src/main/java/com/ace/alfox/api/MapApi.java]: Yes! Great point

File: src/main/java/com/ace/alfox/api/MapApi.java (hunk 2 above)
Comment (coltin) [reply src/main/java/com/ace/alfox/api/MapApi.java:40]: Very temporary ;)

**Pull request merged**